nni/common/serializer.py

```diff
@@ -403,6 +404,8 @@ def load(string: Optional[str] = None, *, fp: Optional[Any] = None, ignore_comme
         JSON string to parse. Can be set to none if fp is used.
     fp : str
         File path to load JSON from. Can be set to none if string is used.
+    preserve_order: bool
+        Use ``OrderedDict`` instead of ``dict``. Note that the order is always preserved even when this is False.
```

Comment:
maybe a little confused when I use `dump`, `load` to serialize models, what does `preserve_order` mean

Reply:
@ultmaster Any idea?

Reply:
Add a link to json-tricks.

Users will not use it if they don't know what that means.